docusaurus/docs/recipes/recipes/subnet-free-ip-monitoring.md

```diff
@@ -0,0 +1,80 @@
+# Monitoring Free IP in customers subnet
+
+In this recipe we show you how to setup monitoring stack to monitor for available IPs in the subnet.
+
+We will be setting up a stack using [AWS Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/) to create a Lambda, cloudwatch dashboard and a cloudwatch alarm for monitoring available free IPs in the subnet.
+
+:::note
+    This guide will take approximately 30 minutes to complete.
+:::
+## Infrastructure
+In the following section we will be setting up the infrastructure for this recipe. 
+
+The lambda deployed here will be calling EC2 APIs at an interval and will emitting free IP metrics to [Cloudwatch Metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/working_with_metrics.html).
+
+### Prerequisites
+
+* The AWS CLI is [installed](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) and [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) in your environment.
+* The [AWS CDK Typescript](https://docs.aws.amazon.com/cdk/latest/guide/work-with-cdk-typescript.html) is installed in your environment.
+* Node.js and Go.
```

Comment:
Why is Go required? Do we need python?

Reply:
yeah we don't need either Go and Python. removed it. Python is already part of the lambda environment.